.travis.yml

```diff
@@ -1,10 +1,10 @@
 sudo: required
-dist: xenial
+dist: bionic
 language: generic
 
 env:
   global:
-    - ROS_DISTRO=kinetic
+    - ROS_DISTRO=melodic
```

Comment:
Instead changing the CI, why you don't maintain melodic and kinetic as kinect is to an Ubuntu LTS? Thanks.

Reply:
well, this is related to: https://github.com/ros-industrial/industrial_ci/issues/406 , apparently archive.ubuntu.com is throttled and CI takes over 50 min to perform, which results in a timeout build failed (for free accounts). Melodic dependencies have cleaned up and CI takes about half of the time to perform (apt is the bottleneck). So this PR is a workaround for CI tests to show green. As for now main repo of rosplan_demos and some PR are failing due to this issue

Reply:
Thanks for the explanation.